test/run-k8s-integration-ci.sh

```diff
@@ -31,8 +31,10 @@ make -C "${PKGDIR}" test-k8s-integration
 
 base_cmd="${PKGDIR}/bin/k8s-integration-test \
             --run-in-prow=true --service-account-file=${E2E_GOOGLE_APPLICATION_CREDENTIALS} \
-            --do-driver-build=${do_driver_build} --teardown-driver=${teardown_driver} --boskos-resource-type=${boskos_resource_type} \
-            --storageclass-files=sc-standard.yaml,sc-balanced.yaml,sc-ssd.yaml --snapshotclass-file=pd-volumesnapshotclass.yaml \
+            --do-driver-build=${do_driver_build} --teardown-driver=${teardown_driver} \
+            --boskos-resource-type=${boskos_resource_type} \
+            --storageclass-files=sc-standard.yaml,sc-balanced.yaml,sc-ssd.yaml,sc-regional.yaml \
```

Comment:
nit: Set this flag in the `[ -z "$gce_region" ]` branch so it's more explicit that `sc-regional.yaml` is only expected to work in regional clusters

Reply:
great point, thanks.